doc/help/configuring.asciidoc

```diff
@@ -120,7 +120,7 @@ c.completion.shrink = True
 ----
 
 Note that qutebrowser does some Python magic so it's able to warn you about
-mistyped config settings. As an example, if you do `c.tabs.possition = "left"`,
+mistyped config settings. As an example, if you do `c.tabs.position = "left"`,
```

Comment:
This one is deliberate, see the context around it.

Reply:
This one still needs to be undone.